src/xrNetServer/empty/NET_Client.cpp

```diff
@@ -175,7 +175,7 @@ XRNETSERVER_API bool psNET_direct_connect = false;
  ****************************************************************************/
 
 static HRESULT WINAPI Handler(PVOID pvUserContext,
-    DWORD dwMessageType,
+    u32 dwMessageType,
```

Comment:
WinAPI function, should use WinAPI types.
```suggestion
    DWORD dwMessageType,
```